video/video.ino

```diff
@@ -130,9 +131,20 @@ void do_keyboard() {
 	if (getKeyboardKey(&keycode, &modifiers, &vk, &down)) {
 		// Handle some control keys
 		//
-		switch (keycode) {
-			case 14: setPagedMode(true); break;
-			case 15: setPagedMode(false); break;
+		if (down) {

```

Comment:
added the check for `down` in here, as otherwise you end up getting a second execution when the control key is released and the corresponding "up" event is sent